lib/roots_web/schema/cookbook_types.ex

```diff
@@ -27,4 +27,13 @@ defmodule RootsWeb.Schema.CookbookTypes do
       resolve(&Resolvers.CookbookResolver.list/3)
     end
   end
+
+  object :cookbook_mutations do
+    @desc "Create a new cookbook"
+    field :create_cookbook, type: :cookbook do
+      arg(:title, non_null(:string))
```

Comment:
This is fine for now that it's getting created without a user, we just can't demo the create cookbook functionality without authentication 🤷🏽‍♀️